### DISCUSSION TITLE
Hyperlink is not saved

### DISCUSSION THREAD

**Discussion by r-vorlicek** [phase1geo/Minder#264] [Q&A]
Hello.
I have a problem with saving hyperlink attached to node. When I add link via "Add link" in context menu it is not writen to file. It is working but when I restart program it disappear. I cannot find recorded issue.

docs: https://github.com/phase1geo/Minder/wiki/Adding-URL-Links
version: 1.11.2
os: Fedora 33

Pls, where can I find xsd for .minder file? 

Thank you.

Robert Vorlicek

**Comment by phase1geo** [comment]
Minder does not currently have an XSD for its XML file format.

As for the issue of having issues with links, could you create anGitHub issue for this bug?

**Reply by r-vorlicek** [reply]
Created as https://github.com/phase1geo/Minder/issues/265

Thank you
